### ISSUE TITLE
 Scheduler/jobs/Run Log is empty (oracle) when sys schema is not visible

### ISSUE THREAD

**Issue by majkyyx** [dbeaver/dbeaver#17909]
### Description

 Scheduler/jobs/Run Log is empty (oracle) when sys schema is not visible.

Run log show "No Data" label, everytime when sys schema is not visible in database navigator. Only oracle problem.

### DBeaver Version

CE 22.2.1.202209251341

### Operating System

ubuntu, windows 10

### Database and driver

ojdbc8.jar

### Steps to reproduce

Use filter for schema list in database navigator and hide somehow **SYS** schema. If **SYS** schema is not visible, run log is always empty.

Issue of origin https://github.com/dbeaver/dbeaver/issues/14528

To info @LonwoLonwo 

![image](https://user-images.githubusercontent.com/14957734/195531183-08ec7ca5-e181-43e6-af84-d229bc8e091b.png)
![image](https://user-images.githubusercontent.com/14957734/195531198-8f2ce019-822a-4c8c-b26a-14b06a3b1168.png)

**Comment by LonwoLonwo** [comment]
Thanks for the report @majkyyx

**Comment by gzone156** [comment]
Any news on that issue?

**Comment by LonwoLonwo** [comment]
Hello @gzone156 

You always have a workaround.
You can click on your connection -> choose navigation view -> choose "advanced".
Then you will see the SCHEDULER_JOB view.

**Comment by majkyyx** [comment]
> Hello @gzone156
> 
> You always have a workaround. You can click on your connection -> choose navigation view -> choose "advanced". Then you will see the SCHEDULER_JOB view.

Yeah, and  "sys" schema have to be visible - so disable schema filter ;-)

**Comment by gzone156** [comment]
> > Hello @gzone156
> > You always have a workaround. You can click on your connection -> choose navigation view -> choose "advanced". Then you will see the SCHEDULER_JOB view.
> 
> Yeah, and "sys" schema have to be visible - so disable schema filter ;-)

Done that way, thanks for the suggestion!
